### ISSUE TITLE
Screenshots and new animation

### ISSUE THREAD

**Issue by TobiGr** [TeamNewPipe/website#13]
Because my phone is to small I can't make good screenshots and a new animation.
Besides this, we should discuss which features should be shown (in the screenshots and in the new animation as well).

**Comment by TheAssassin** [comment]
I guess it's possible to record the Android emulator window.

**Comment by TobiGr** [comment]
That's a good idea.

**Comment by TheAssassin** [comment]
I'm working on this right now. I'm creating a basic version of something similar to what we have, but I'd like to have some video that showcases the screen rotation behavior etc.

**Comment by TheAssassin** [comment]
*Offtopic:*
By the way, @theScrabi and I decided to test a FreeNode IRC channel for communication. You can join it if you want, and chat with others there (well, I hope others will join it, too).

**Comment by TheAssassin** [comment]
*More offtopic:*
The channel name is [#newpipe](https://webchat.freenode.net/?channels=%23newpipe).

**Comment by TheAssassin** [comment]
I tried to make a video using the video recording built into Android Studio. The resulting video has a very low framerate, and doesn't look very good. I guess one has to use a normal screen recorder or just record on a real device.

**Comment by theScrabi** [comment]
There is a screenrecorder embedet into cyanogenmod. I give it a shot.

**Comment by ponypower** [comment]
Hey guys, any results yet?

**Comment by TheAssassin** [comment]
Not yet. I was waiting for @theScrabi to tell us about his results with the CyanogenMod screen recorder.

**Comment by TobiGr** [comment]
Is there any progress?

**Comment by theScrabi** [comment]
Ow yea I worked on a video, but so far I don't like that.

Sorry I was still in the state of not using a video.

**Comment by mauriciocolli** [comment]
Hey, I did a screen record [here on Google Drive](https://drive.google.com/open?id=0B4_v6DLdrpy3NnBscFVUN255cUE) (please download it, as the Google player's compression ruined the video and the frame rate).

It's not the final, just to give an idea and here's some details:
- The first and last frame are very similar, giving the seamless effect in a video with loop enabled
- 60fps because makes the video smoooth as f
- Show the "principal" characteristics of NewPipe

Of course it's not perfect, so I would appreciate any input of what could be done to improve the "showcase" video.

**Comment by theScrabi** [comment]
It's good, although
- channel support is not shown
- and I wonder why playing regular videos is not shown

**Comment by TheAssassin** [comment]
@theScrabi to make it easier for @mauriciocolli and others to record videos and contribute them, you should publish a list of requirements (what has to be shown in what way).

**Comment by theScrabi** [comment]
Ok things to show would be:
 - Search Video
 - Show channel
 - play Video
 - download video/music
 - show download screen (while downloading, also there should be some videos/music downloaded already)
 - show play in background
 - show play as popup

**Comment by wb9688** [comment]
I've tried to create [a video](https://wb9688.stackstorage.com/s/XilyIREE6PRQOza), but it's probably too fast.

**Comment by theScrabi** [comment]
Yea it's a bit fast.

**Comment by TheAssassin** [comment]
I have recently discovered a [new app](https://f-droid.org/repository/browse/?fdid=com.orpheusdroid.screenrecorder) for screen recording, if anyone wants to give it a try, it works quite well.

**Comment by TobiGr** [comment]
To get the website a little more forward I created a video. You need to [download ](
https://mega.nz/#!YWYxDRAS!KORLvgS9nCOuru9IURiLjmQKBWsR16D2izYVpXgzKTQ
)it. @theScrabi Can you have a look at it and tell me whether it is too fast or something important is missing?

**Comment by theScrabi** [comment]
@TobiGr It's good, the only thing that is missing is the tab indicator which you can enable in the debug settings. It's sometimes hard to follow where you touched the screen.

**Comment by TobiGr** [comment]
@theScrabi That's right. I'll do a new one over week. I thought about skipping the options in the download dialog to make the video a little shorter.

**Comment by theScrabi** [comment]
Yes thats ok :)
